### PR TITLE
Cleaning up the README

### PR DESCRIPTION
**Pull request by nuwand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,52 +57,86 @@ WSO2 API Microgateway acts as a proxy that is capable of performing security val
...
+factors and so on.
+The WSO2 API Microgateway is designed to expose heterogeneous microservices as APIs to end consumers using a common API 
+interface based on the Open API Specification. This helps expose microservices using a unified interface to external 
+consumers, internal consumers and partners. It applies the common quality of service attributes on API requests such as 
+security, rate limiting and analytics and also offers a wide range of features which helps organizations to deploy APIs 
+microservice architectures efficiently.
 
 #### Microgateway quick start
-Lets see how we can expose pet store API using microgatway using the open API definition
+Let host our first API on a Microgateway.
```
Comment (fazlan-nazeem) [inline README.md]: Let's?

File: README.md [hunk 2]
```diff
@@ -47,52 +57,86 @@ WSO2 API Microgateway acts as a proxy that is capable of performing security val
...
 ```
-docker run -d -v <PROJECT_TARGET_PATH>:/home/exec/ -p 9095:9095 -p 9090:9090 -e project="petstore-project"  wso2/wso2micro-gw:3.0.1
+docker run -d -v <PROJECT_TARGET_PATH>:/home/exec/ -p 9095:9095 -p 9090:9090 -e project="petstore"  wso2/wso2micro-gw:3.0.1
 
-<PROJECT_TARGET_PATH> - The path of the target directoy created inside the project directory
+<PROJECT_TARGET_PATH> - The path of the target directory created inside the project directory
 
-Note: We actually need to mount the file created with .balx extension into the docker imgage. The target dirctory contains other generated files not required for docker image. 
-So we can copy the .balx file to seperate directory and mount that directory
+Note: We actually need to mount the file created with .balx extension into the docker imgage. The target dirctory
```
Comment (mushthaq33) [inline README.md]: Need to change to directory

File: README.md (hunk 1 above)
Comment (nuwand) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (nuwand) [reply README.md]: Fixed.

**Pull request merged**